docs/databases.md

```diff
@@ -0,0 +1,55 @@
+# Firebase integration
```

Comment:
I think it would be good to have a very short why Firebase was chosen for the database integration.